[2024-08-22T11:00:23.159+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-22T11:00:23.176+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_py_dependencies_v1.get_sk manual__2024-08-22T11:00:22.809879+00:00 [queued]>
[2024-08-22T11:00:23.186+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_py_dependencies_v1.get_sk manual__2024-08-22T11:00:22.809879+00:00 [queued]>
[2024-08-22T11:00:23.187+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-22T11:00:23.203+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_sk> on 2024-08-22 11:00:22.809879+00:00
[2024-08-22T11:00:23.210+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=231) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-22T11:00:23.210+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_py_dependencies_v1', 'get_sk', 'manual__2024-08-22T11:00:22.809879+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_dependencies.py', '--cfg-path', '/tmp/tmp_5t9o0p6']
[2024-08-22T11:00:23.213+0000] {standard_task_runner.py:72} INFO - Started process 232 to run task
[2024-08-22T11:00:23.214+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask get_sk
[2024-08-22T11:00:23.264+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_py_dependencies_v1.get_sk manual__2024-08-22T11:00:22.809879+00:00 [running]> on host a66aa792e14d
[2024-08-22T11:00:23.349+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='matea' AIRFLOW_CTX_DAG_ID='dag_with_py_dependencies_v1' AIRFLOW_CTX_TASK_ID='get_sk' AIRFLOW_CTX_EXECUTION_DATE='2024-08-22T11:00:22.809879+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-22T11:00:22.809879+00:00'
[2024-08-22T11:00:23.350+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-22T11:00:23.855+0000] {logging_mixin.py:190} INFO - 1.5.0
[2024-08-22T11:00:23.856+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-22T11:00:23.863+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-22T11:00:23.864+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_py_dependencies_v1, task_id=get_sk, run_id=manual__2024-08-22T11:00:22.809879+00:00, execution_date=20240822T110022, start_date=20240822T110023, end_date=20240822T110023
[2024-08-22T11:00:23.889+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-22T11:00:23.902+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-22T11:00:23.903+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
